bloom_nofos/nofos/api/schemas.py

```diff
@@ -0,0 +1,77 @@
+from ninja import ModelSchema, Schema
+from typing import List, Optional
+from nofos.models import Nofo, Section, Subsection
+
+
+class SubsectionSchema(ModelSchema):
+    class Config:
+        model = Subsection
+        model_fields = [
+            "name",
+            "html_id",
+            "order",
+            "tag",
+            "body",
+            "callout_box",
+            "html_class",
+        ]
+        model_fields_optional = ["html_class"]  # Fields that should have defaults
+
+
+class SectionSchema(ModelSchema):
+    subsections: List[SubsectionSchema]
+
+    class Config:
+        model = Section
+        model_fields = ["name", "html_id", "order", "has_section_page"]
+
+
+class NofoBaseSchema(ModelSchema):
+    class Config:
+        model = Nofo
+        model_fields = [
+            "id",
+            "title",
+            "filename",
+            "short_name",
+            "number",
+            "opdiv",
+            "agency",
+            "tagline",
+            "application_deadline",
+            "subagency",
+            "subagency2",
+            "author",
+            "subject",
+            "keywords",
+            "theme",
+            "cover",
+            "icon_style",
+            "status",
+            "cover_image",
+            "cover_image_alt_text",
+            "inline_css",
+        ]
+        model_fields_optional = [
+            "subagency",
+            "subagency2",
+            "author",
+            "subject",
+            "keywords",
+            "cover_image",
+            "cover_image_alt_text",
+            "inline_css",
+        ]
+
+
+class NofoSchema(NofoBaseSchema):
+    sections: List[SectionSchema]
+
```

Comment:
Looks like this solves the ordering problem! How does this work (and how did you figure it out)?

Reply:
Completely by accident. Glad it worked though